pkgs/tools/misc/mommy/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, writeText
+, shellspec
+  # usage:
+  # pkgs.mommy.override {
+  #  mommySettings.sweetie = "catgirl";
+  # }
+  #
+  # $ mommy
+  # who's my good catgirl~
+, mommySettings ? null
```

Comment:
Are there any other packages that configure settings like this? I'm not sure how useful this would be since it requires people to check the `default.nix` of mommy.

Reply:
firefox comes with a similar wrapper ([`extraPerfs`](https://github.com/ckiee/nixpkgs/blob/8584d6f5f83db046413c9c3acfd83d1e47df84bf/pkgs/applications/networking/browsers/firefox/wrapper.nix#L41-L45)), and it's in `mommy`'s spirit to be `nix copy`-able.